README.md

```diff
@@ -6,6 +6,8 @@ We’ve designed the system to make it easy to share packages on services like G
 
 Swift Package Manager includes a build system that can build for macOS and Linux. Xcode 11 integrates with libSwiftPM to provide support for iOS, watchOS, and tvOS platforms.
 
+The [SourceKit-LSP](https://github.com/apple/sourcekit-lsp) project leverages libSwiftPM and provides Language Server Protocol implementation for various [editors](https://github.com/apple/sourcekit-lsp/tree/master/Editors).
```

Comment:
Maybe we should just say "for editors that support LSP" instead of linking to that page?  I don't want to imply that we only support the editors explicitly listed on that page.  Otherwise LGTM.